frontend/src/action/transaction.js

```diff
@@ -3,9 +3,9 @@ export function deposit(data) {
         type: 'DEPOSIT',
         payload: {
             request: {
-                method: 'post',
+                method: 'patch',
```

Comment:
It's better post to be used for deposit as it creates a new accounting operation. Keep in mind that Name of the PR doesn't have information information regarding these changes.